src/iterator.js

```diff
@@ -61,18 +57,112 @@ class Iterator {
     return this.setLowerIntervals(intervalTime, intervalTime);
   }
 
+  calculateFortnight(intervalTime) {
+    // calculate the difference in weeks between the dtStart
+    // and the current date.
+    const weekDiff = moment(this.rule.dtStart.date).diff(intervalTime.date.toISOString(), 'week');
```

Comment:
the `diff`, `add` and `isBefore` should really move to the TimezoneDate class, rather than all the 

`moment(interval.date).add` it would just be `interval.add`